src/backend/gl/src/window/egl.rs

```diff
@@ -114,11 +115,9 @@ impl Inner {
             return Err(hal::UnsupportedBackend);
         }
         let display_ext_list = display_extensions.split_whitespace().collect::<Vec<_>>();
-        let required_display_extensions = ["EGL_KHR_create_context", "EGL_KHR_surfaceless_context"];
-        for required_ext in required_display_extensions.iter() {
-            if !display_ext_list.contains(required_ext) {
-                log::warn!("{} is not present", required_ext);
-            }
+        if !display_ext_list.contains(&"EGL_KHR_create_context") {
```

Comment:
do we actually require this? My impression was that we could get away even without it